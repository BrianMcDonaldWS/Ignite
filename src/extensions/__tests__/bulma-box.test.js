import MarkdownIt from 'markdown-it';
import box from '../bulma-box';

const md = new MarkdownIt().use(box);

test('bulma - box', () => {
  expect(
    md.render(`
::: box

# Title

I can contain various combinations of elements.
:::
  `)
  ).toMatchSnapshot();
});
