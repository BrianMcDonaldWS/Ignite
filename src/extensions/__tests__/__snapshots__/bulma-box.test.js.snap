// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - box 1`] = `
"
        <div class=\\"column \\">
          <div class=\\"box\\">
      <h1>Title</h1>
<p>I can contain various combinations of elements.</p>

        </div>
      </div>
    "
`;
